src/rt/sections_darwin_64.d

```diff
@@ -0,0 +1,178 @@
+/**
+ * Written in the D programming language.
+ * This module provides Darwin 64 bit specific support for sections.
+ *
+ * Copyright: Copyright Digital Mars 2016.
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors: Jacob Carlborg
+ * Source: $(DRUNTIMESRC rt/_sections_darwin_64.d)
+ */
+module rt.sections_darwin_64;
+
+import core.sys.darwin.mach.dyld;
+import core.sys.darwin.mach.getsect;
+import core.sys.posix.pthread;
+
+version (OSX)
+    version = Darwin;
+else version (iOS)
+    version = Darwin;
+else version (TVOS)
+    version = Darwin;
+else version (WatchOS)
+    version = Darwin;
+
+version (Darwin):
+version (D_LP64):
```

Comment:
The commit message says "both x86-64 and ARM", but I assume you mean `s/ARM/AArch64/` ? `version (ARM)` is 32 bits only.

Reply:
Yes, Apple calls AArch64 ARM64 and `arm64-apple-ios12.0` is a valid target triple in Clang/LDC.